src/components/core/geoChart/tests/GeoChartOptionsWrapper.spec.tsx

```diff
@@ -429,6 +432,68 @@ describe("GeoChartOptionsWrapper", () => {
         });
     });
 
+    it("should return props with new excutionResult be replaced emptyHeaderString", async () => {
+        const executionResult: Execution.IExecutionResult = {
+            data: [],
+            paging: {
+                count: [1, 1],
+                offset: [0, 0],
+                total: [1, 1],
+            },
+            headerItems: [
+                [
+                    [
+                        {
+                            attributeHeaderItem: {
+                                name: "",
+                                uri: "/gdc/md/storybook/obj/694/elements?id=1",
+                            },
+                        },
+                    ],
+                ],
+            ],
+        };
+        const props: Partial<IGeoChartInnerProps> = {
+            config: {
+                ...getGeoConfig({
+                    isWithLocation: true,
+                }),
+                colorPalette: DEFAULT_COLOR_PALETTE.slice(0, 5),
+            },
+            execution: {
+                executionResponse: getExecutionResponse(true),
+                executionResult,
+            },
+            dataSource: locationDataSource,
+        };
+        const wrapper = renderComponent(props);
+        const executionString = "execution";
+        expect(wrapper.props()[executionString]).toEqual(
```

Comment:
should we use it directly?
`expect(wrapper.prop("execution").executionResult.headerItems).toEqual([ ... ]);`

Reply:
Typescript got error with 

> wrapper.prop("execution").executionResult.headerItems